api/src/pcapi/core/mails/CONTRIBUTING.md

```diff
@@ -1,4 +1,5 @@
-# Envoyer des mails dans pcapi
+# Envoyer des mails dans pcapi 
+Mailjet - Obsolète depuis 2022-05-01
```

Comment:
Est ce qu'on veut vraiment le mentionner dans la doc, vu qu'il n'y a plus de notion "Mailjet" dans le code ?

Reply:
Le temps que la documentation soit mis à jour avec Sendinblue

Reply:
en attendant la màj de la documentation qui fera l'objet d'un autre ticket, why not ?

Reply:
c'est la notion de mail qui est importante

Reply:
Puisque tu es sur le sujet, autant mettre à jour la documentation en même temps, non ? Au moins, ce sera fait. :)

Reply:
yes done